### PR TITLE
Cache Ext works with multiple Extensions

### PR DESCRIPTION
**Pull request by ssnickolay**

### CODE REVIEW

File: lib/graphql/cache/resolver.rb [hunk 1]
```diff
@@ -11,8 +11,8 @@ def initialize(type, field)
         @field = field
       end
 
-      def call(obj, args, ctx, &block)
-        resolve_proc = proc { block.call(obj, args, ctx) }
+      def call(obj, args, ctx, block)
+        resolve_proc = block #proc { block.call(obj, args, ctx) }
```
Comment (anhang) [inline lib/graphql/cache/resolver.rb:15]: is this change intentional?

**Review by anhang — APPROVED** [review]

File: lib/graphql/cache/resolver.rb (hunk 1 above)
Comment (ssnickolay) [reply lib/graphql/cache/resolver.rb:15]: Yep, I left it because last two PRs added bunch of functionality and I think if I work on `{keep_fresh: true}`, I'll refactor new files a little bit.
